perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2024 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+# limitations under the License.
+# To run the script, run in terminal :
+# python3 generate_folder_and_files.py <config-file.json>
+
+import argparse
+import json
+from datetime import datetime as dt
+import logging
+import sys
+import subprocess
+from subprocess import Popen
+
+OUTPUT_FILE = str(dt.now().isoformat()) + '.out'
+
+logging.basicConfig(
```

Comment:
Why are we using this instead of `logger.error(message) ` to log error message?

Reply:
We are redirecting logs to output file so that it can be saved for future reference in case of failure while creating very large directories